nubis/puppet/files/run-scout

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash -l
 
+# Sleep for a random amount of time within 10 minutes
+# Startups are rough and we should make it sleep for a bit to let
+# the host settle down before running Scout
+sleep $(( $RANDOM % 600 ))
```

Comment:
sleep $(( $RANDOM % 600 ))
          ^-- SC2004: $/${} is unnecessary on arithmetic variables.

Means sleep $(( RANDOM % 600 )) is sufficient